packages/api/src/server.ts

```diff
@@ -11,6 +11,9 @@ import { initLocales } from './i18n'
 import { initEntities } from './entities'
 import { analyzeTwitterFollowings } from './rpcs'
 import { clearCache } from './caching'
+import { test } from '@stateofjs/common/src/index'
```

Comment:
@SachaG here's an example of the common module import. It can be cleaned up to get rid of the `src` in the path, but it's more of an effort than I'm ready to invest in right now. If you want to just lift TS modules up into the root `/packages/common` directory that would be a valid compromise.